docs/DeveloperGuide.md

```diff
@@ -348,20 +362,29 @@ Getting the list of inpatients and outpatients involves the following steps:
 If additional parameters are inputted (e.g. `get /inp hello world`), the extra parameters will be ignored, similar to 
 how `help`, `list`, `exit` and `clear` are executed.
 
-The Sequence Diagram below illustrates the interactions within the `Logic` component for the `execute("get /inp")` API call.
-
-![GetInpatientSequenceDiagram](images/GetInpatientSequenceDiagram.png)
-
 #### Getting the past appointments of a patient (`/appt`)
 
 Getting the past appointments of a patient involves the following steps:
 1. prefix `/appt` is matched using an instance of `GetCommandParser`
-2. a new `GetPastAppointmentCommandParser` instance is created and parses the user input (specificallly the index inputted)
+2. a new `GetPastAppointmentCommandParser` instance is created and parses the user input (specifically the index inputted)
 3. a `GetPastAppointmentCommand` instance containing the index of the patient to be updated is created and returned
-4. the `GetPastAppointmentCommand` command is executed, accessing the list of `PastAppointment` of the specified patient
+4. the `GetPastAppointmentCommand` is executed, accessing the list of `PastAppointment` of the specified patient
    to be returned in a `CommandResult`
 5. The list of `PastAppointment` will then be displayed in the `ResultDisplay`
 
+#### Getting patients with an appointment on a specified date (`get /appton`)
+
+Getting patients with an appointment on a specified date involves the following steps:
+1. prefix `/appton` is matched using an instance of `GetCommandParser`
+2. a new `GetAppointmentByDateCommandParser` instance is created and parses the user input (specifically the date inputted)
+3. a `GetAppointmentByDateCommand` instance containing the date of the appointment is created and returned
+4. the `GetAppointmentByDateCommand` command is executed, accessing the list of `PastAppointment` of the specified patient
```

Comment:
Perhaps the word "command" following `GetAppointmentByDateCommand` can be removed for consistency with point 4 of `get /appt`.